src/app/modules/main/communities/view.nim

```diff
@@ -567,8 +567,7 @@ QtObject:
   proc cancelRequestToJoinCommunity*(self: View, communityId: string) {.slot.} =
     self.delegate.cancelRequestToJoinCommunity(communityId)
 
-  proc requestCommunityInfo*(self: View, communityId: string, importing: bool) {.slot.} =
-    # TODO update the slot to accept the shard arguments when it's available from the QML
+  proc requestCommunityInfo*(self: View, communityId: string, shardCluster: int, shardIndex: int, importing: bool) {.slot.} =
     self.delegate.requestCommunityInfo(communityId, shardCluster = -1, shardIndex = -1, importing)
```

Comment:
please update the arguments here as well 😄

Reply:
hm.. did I pick up an old commit hash 😮